great_expectations/core/usage_statistics/schemas.py

```diff
@@ -23,31 +23,55 @@
                 "anonymized_name": {"$ref": "#/definitions/anonymized_string"},
                 "parent_class": {"type": "string", "maxLength": 256},
                 "anonymized_class": {"$ref": "#/definitions/anonymized_string"},
-                "sqlalchemy_dialect": {"type": "string", "maxLength": 256},
             },
-            "additionalProperties": False,
-            "required": ["parent_class", "anonymized_name"],
+            "additionalProperties": True,
+            # we don't want this to be true, but this is required to allow show_cta_footer
+            # Note AJB-20201218 show_cta_footer was removed in v 0.9.9 via PR #1249
+            "required": ["parent_class"],
```

Comment:
Note : these lines were moved here from another section. Don't worry about this :)